CONTRIBUTING.md

```diff
@@ -167,13 +180,17 @@ any PMC member:
 1. Merge the release proposal PR into the master branch of the repository. There
    should not be any conflicts as the text in the files that have changed should
    only be changed during this release process.
-1. Create a new branch called **"v$major.$minor"** from the **Release SHA**.
+1. Checkout the **Release SHA** (either to a local branch or in 'detached HEAD'
```

Comment:
@duglin You're right that we could just lock master instead of doing this, but I'm easy either way